### PR TITLE
A handful of table improvements.

### PR DESCRIPTION
**Pull request by vext01**
Bold entries in permutation grid, rename PyHyp_{c'}, start counting from one.

### CODE REVIEW

File: latex_table.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "HippyVM": "HippyVM",
     "PyHyp-mono": "PyHyp$_m$",
     "PyHyp-comp": "PyHyp$_c$",
-    "PyHyp-comp-rev": "PyHyp$_{c'}$",
+    "PyHyp-comp-rev": "PyHyp$_{c2}$",
```
Comment (ltratt) [inline latex_table.py:20]: Can we call this PyHyp$_r$ or something more descriptive than `c2`?

File: latex_table.py [hunk 2]
```diff
@@ -250,14 +250,18 @@ def process_db_perms_data(results):
...
     if val is None: # no result for that combo
         return ""
     else:
         err_s = "\\pm %.4f" % err if err is not None else ""
-        return ("$\substack{\\mathmakebox[%s][r]{%.3f}%s\\\\"
+        val_s = "%.3f" % val
+        if bold:
+            err_s = "\\mathbf{%s}" % err_s
+            val_s = "\\mathbf{%s}" % val_s
+        return ("$\substack{\\mathmakebox[%s][r]{%s}%s\\\\"
```
Comment (ltratt) [inline latex_table.py:262]: Why the brackets around the return? It's not returning a tuple I don't think.

File: latex_table.py (hunk 1 above)
Comment (vext01) [inline latex_table.py:20]: If you like. However a few months ago you expressed dislike for the term "reversed".

File: latex_table.py (hunk 2 above)
Comment (vext01) [inline latex_table.py:262]: It's a way of breaking a long string without using a binary `+`.

File: latex_table.py (hunk 1 above)
Comment (ltratt) [inline latex_table.py:20]: Ah, yes, I remember. We could maybe call it "inverse" or something? Or maybe `c'` is as good as it gets...

Comment (vext01) [inline latex_table.py:20]: I don't mind too much. Inverse has the same implied meaning as reverse. I find `c'` concise. You choose.

**Pull request merged**